storybook/stories/ComponentLists.tsx

```diff
@@ -41,9 +48,9 @@ export default () =>
         ComponentList={DAOs}
         Component={DAO}
         RequiredContext={(props) => (
-          <Arc config={arcConfig}>
-          {props.children}
-          </Arc>
+          <Loader render={<Test />}>
```

Comment:
Same as before - Remember it's just in daos list component

Reply:
Same as previous comments, let's change render to be like formik and react-router do:
```
<Loader render={(props: LoaderProps) => (
  <div>{props.errors.map(error => error)}</div>
)}>
```